packages/realm-server/tests/realm-server-test.ts

```diff
@@ -670,4 +685,11 @@ module('Realm Server serving from root', function (hooks) {
       'the directory response is correct'
     );
   });
+
+  test('serves a root directory GET request that results in redirect', async function (assert) {
+    let response = await request2.get('http://127.0.0.1:4446/demo');
```

Comment:
Getting a `Error: ECONNREFUSED: Connection refused` here. I'm trying to have a realm where the root url is `/demo/` - am I specifying this right in `testRealmServer2`?

Reply:
realms always end in a trailing slash, your URL example above does not end in a trailing slash (a trailing slash signifies the entry is a directory).

Reply:
Introduced a new module for this